cgrok/grok.c

```diff
@@ -1,7 +1,4 @@
 #include "grok.h"
-#include <dlfcn.h>
```

Comment:
Including `dlfcn.h` on Windows is a nightmare, and we will never be `dlopened` so I ripped this out.